docs/setup-nginx-proxy-with-minio.md

```diff
@@ -36,19 +45,20 @@ server {
 }
 ```
 
-Note:
+#### 3.1.4 Modify the host settings in the configuration file:
+* `example.com`: Specifies the host name. Replace this with the name of your own host.
+* `http://localhost:9000`: Specifies the server name. Replace this with the name of your own server.
 
-* Replace example.com with your own hostname.
-* Replace ``http://localhost:9000``  with your own server name.
-* Add ``client_max_body_size 1000m;`` in the ``http`` context in order to be able to upload large files — simply adjust the value accordingly. The default value is `1m` which is far too low for most scenarios. To disable checking of client request body size, set ``client_max_body_size`` to `0`.
-* Nginx buffers responses by default. To disable Nginx from buffering Minio response to temp file, set `proxy_buffering off;`. This will improve time-to-first-byte for client requests.
-* Nginx disallows special characters by default.  Set ``ignore_invalid_headers off;`` to allow headers with special characters.
+#### 3.1.5 (Optional) Review additional settings in the configuration file:
+* `client_max_body_size`: Specifies the maximum size of the client request body. Set this value to `1000m` in the `http` context to enable large file uploads. This overrides the default value of `1m` which is too low for most scenarios. Set this value to `0` to disable checking the size of the client request body.
+* `proxy_buffering`: Enables or disables buffering responses to a temporary file. Set this value to `off` to disable buffering and improve time-to-first-byte for client requests. This setting is enabled by default.
+* `ignore_invalid_headers`: Allows or disallows special characters. Set this value to `off` to allow headers with special characters. This setting disallows special characters by default.
```

Comment:
`ignore_invalid_headers` is not natively in the config file, nginx just blocks special chars in headers as the default behavior, should change to reflect this

Reply:
@deekoder , @eco-minio : this is resolved in the latest commit. Please see https://github.com/jamesneimanconsulting/cookbook/blob/patch-6/docs/setup-nginx-proxy-with-minio.md.